src/main/java/org/example/model/builder/UserBuilder.java

```diff
@@ -0,0 +1,35 @@
+package org.example.model.builder;
+import org.example.model.Observer.DesiredPath;
+import org.example.model.User;
+public class UserBuilder {
+        private Integer id;
```

Comment:
You need to fix the formatting. Just go to the file and type command + option + L if you're using the Mac OS and Ctrl + Alt + L if you're using the either Windows or Linux OS.